api/views.py

```diff
@@ -22,3 +22,18 @@ def post(self, request):
             return Response(status=status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class CustomerList(APIView):
+    authentication_classes = [TokenAuthentication, HostAuthentication, SignatureValidation]
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        serializer = CustomerSerializer(data=request.data)
+        if serializer.is_valid():
+            customer = serializer.validated_data
+            check_customer_is_partner.delay(customer)
```

Comment:
I find this task name a bit misleading because it's not only checking but also updating the Wordpress user.

Following https://github.com/coopdevs/lazona_connector/pull/63/files#r632564778, I'd rather name this task `create_customer` and so the task becomes the true entry point like a view method in a regular Django app.

Then, within `create_customer` we can call whatever classes we need to implement the use case.

Reply:
Totally right.